### DISCUSSION TITLE
Was v1.65.2 pulled?

### DISCUSSION THREAD

**Discussion by aptalca** [gitpod-io/openvscode-server#333] [General] [answered]
The release for v1.65.2 disappeared from GitHub. Was it pulled?

Thanks

**Comment by filiptronicek** [comment, marked as answer]
Hi there @aptalca, thanks for bringing this to our attention. We are still trying to figure out why the release isn't appearing in the releases tab, but until then, it is still available at https://github.com/gitpod-io/openvscode-server/releases/tag/openvscode-server-v1.65.2 directly.

**Reply by jeanp413** [reply]
The 1.65.2 release is showing up again in the releases tab

**Comment by aptalca** [comment]
Thanks. Perhaps it was a github glitch as it's showing up now.

The problem was, we, at linuxserver.io, rely on upstream releases to publish our docker images and the glitch caused our ci to build and publish v1.65 as the latest. Oh well. It will retrigger again within the hour and should push 1.65.2 as latest.

**Reply by filiptronicek** [reply]
Sorry for that, we're going to be releasing 1.66 soon, so hopefully that will appear there without any issues 🤞.

**Reply by aptalca** [reply]
No worries. It got picked up by the ci 🥳  https://ci.linuxserver.io/blue/organizations/jenkins/Docker-Pipeline-Builders%2Fdocker-openvscode-server/detail/main/54/pipeline
